protocol/avdl/chat1/wallet.avdl

```diff
@@ -0,0 +1,8 @@
+@namespace("chat.1") protocol wallet {
+
```

Comment:
I'm not sure this type needs to be here, how come you put it in AVDL?

Reply:
thought the server might need it, I can take it out though

Reply:
Yeah don't think so, should just be turned into a normal payment after this step.

Reply:
In fact it is probably worthwhile to just create whatever type the Stellar send code expects for what would have been generated from the CLI.

Reply:
Can make another one